openjdk/excludes/ProblemList_openjdk13-openj9.txt

```diff
@@ -181,11 +181,11 @@ java/lang/management/MemoryMXBean/MemoryTestZGC.sh https://github.com/adoptium/a
 
 # jdk_net
 
-java/net/Inet4Address/PingThis.java	https://github.com/AdoptOpenJDK/openjdk-infrastructure/issues/1127	aix-all
-java/net/MulticastSocket/Promiscuous.java https://github.com/AdoptOpenJDK/openjdk-infrastructure/issues/699 linux-s390x
-java/net/MulticastSocket/PromiscuousIPv6.java	https://github.com/AdoptOpenJDK/openjdk-infrastructure/issues/699	linux-s390x
-java/net/MulticastSocket/SetLoopbackMode.java https://github.com/AdoptOpenJDK/openjdk-infrastructure/issues/699 linux-s390x
-java/net/MulticastSocket/Test.java https://github.com/AdoptOpenJDK/openjdk-infrastructure/issues/699 linux-s390x
+java/net/Inet4Address/PingThis.java	https://github.com/adoptium/nfrastructure/issues/1127	aix-all
+java/net/MulticastSocket/Promiscuous.java https://github.com/adoptium/nfrastructure/issues/699 linux-s390x
+java/net/MulticastSocket/PromiscuousIPv6.java	https://github.com/adoptium/nfrastructure/issues/699	linux-s390x
+java/net/MulticastSocket/SetLoopbackMode.java https://github.com/adoptium/nfrastructure/issues/699 linux-s390x
+java/net/MulticastSocket/Test.java https://github.com/adoptium/nfrastructure/issues/699 linux-s390x
```

Comment:
'i' in infrastructure - best to check all the files in the PR, see more of same in files below

Reply:
Done